Jenkinsfile

```diff
@@ -41,5 +41,5 @@
 //@Library(value="pipeline-lib@your_branch") _
 
 /* groovylint-disable-next-line CompileStatic */
-packageBuildingPipelineDAOSTest(['distros': ['centos7', 'el8', 'el9', 'leap15', 'ubuntu20.04'],
+packageBuildingPipelineDAOSTest(['distros': ['el8', 'el9', 'leap15', 'ubuntu20.04'],
                                 'test-tag': 'pr'])
```

Comment:
Do we know of any tests in DAOS' `src/tests/ftest` that would exercise this change?  If so, we should add it's/their tags to this tag list.  Maybe @tanabarr might know of some tests that would be applicable?

Maybe there are no tests yet because this change needs to be made in order to write some tests that would use this new addition?

Reply:
This PR is required for new tests to be implemented as it is planned by @shimizukko.

Reply:
The test that will use this PR is: https://github.com/daos-stack/daos/pull/13620
It assumes `/usr/share/spdk/scripts/rpc.py` exists in the environment. The test tag is `test_no_activity`